apiservermsgs/common.go

```diff
@@ -15,14 +15,21 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import ()
-
 const PGO_VERSION = "4.3.0"
 
 // Ok status
 const Ok = "ok"
+
+// Error code string
 const Error = "error"
 
+// False is the string boolean, used when checking upgrade status
+const False = "false"
```

Comment:
Per earlier comment, this should be moved to the `config/annotations.go` file in `ANNOTATIONS_FALSE`

Reply:
updated, thanks!